.travis-ci.sh

```diff
@@ -16,7 +16,7 @@ set -o errexit -o nounset
 
 eval `opam config env`
 opam install lmdb --with-test --with-doc --deps-only -v
-dune runtest --force
+dune build --force @runtest @doc tests/bench.exe
```

Comment:
This is ... kinda old. the script for travis ci already deal with testing. The only thing we really need to do is generate the doc if necessary.